include/exadg/postprocessor/output_generator_scalar.cpp

```diff
@@ -113,6 +113,16 @@ OutputGenerator<dim, Number>::setup(dealii::DoFHandler<dim> const & dof_handler_
   }
 }
 
+template<int dim, typename Number>
+void
+OutputGenerator<dim, Number>::setup_after_coarsening_and_refinement(
+  dealii::DoFHandler<dim> const & dof_handler_in,
+  dealii::Mapping<dim> const &    mapping_in)
+{
+  dof_handler = &dof_handler_in;
```

Comment:
compared to the `setup` just above these lines, I did not include the output of the boundary IDs processor ID etc, since this can generate quite some output.

Reply:
Why did you introduce these functions? I thought the DoFHandler and Mapping objects stay alive during adaptive mesh refinement, so why is there a need to reset the pointers?

Reply:
I think we can simply skip the postprocessor->setup_after_coarsening_and_refinement(); since in the present case it does not need to do anything as `dof_handler` and `mapping` are changed, but the pointers still valid. I got confused it seems ...

Reply:
that is true, it was completely useless.